doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+ASM | MA | YES |Run az390 assembler as subtask of mz390 passing BAL.  Note NOASM is for use in pure text processing programs which only use conditional macro code and AREAD/PUNCH with extensions to process ascii text files.  NOASM does not support ordinary symbol attribute tests, OPSYN, or lookahead mode, and requires CHKMAC(0) and CHKSRC(0-2).
+ASSIST | MAE | NO | Enable assembly and execution of 11 new ASSIST extended instructions to simplify I/O for student use.  This option also sets NOLOADHIGH.  See ASSIST Support.
+AUTOLINK | L | YES | Search for unresolved external references in SYSOBJ directory list which defaults to linklib.
+BAL | M | NO | Generate BAL expanded assembler source file
+BS2000 | MA | NO | Support Siemens BS2000 assembler global variables
+CHKMAC(0-2) | M | 0 | Check macros during loading as follows: <br />0 - no checking<br />1- check for missing AGO and AIF labels and issue warning<br />2 - also check missing labels and also check for non comment text after MEND
+CHKSRC(0-3) | MA | 1 | Check input source files as follows:<br />0 - no checking<br />1 - check MLC or BAL input source files for any  non-ASCII characters and issue error<br />2 - check MLC, MAC, CPY, and BAL input source files for any non-ASCII characters and issue error.<br />3 - also check for out of sequence characters in 73-80 or any non-blank characters beyond 80 and issue error.<br />Note this should detect any EBCDIC literal character strings containing non-ASCII characters such as binary 0 byte which must be changed to hex X'00' type strings for portability.
+CICS | MA | NO | Support EXEC CICS pre-processor expansion and constants.  If this option is not on during macro expansion, PROLOG and EPILOG option settings will be ignored.
+CODEPAGE(ascii+ebcdic+LIST) | MALE | YES | The default is CODEPAGE(ISO-8859-1+IBM1047).  If +LIST is added the mapping of the 2 codepages along with printable character and Unicode values are displayed on ERR file.  A hex dump of the tables and a list of the valid ASCII and EBCDIC Unicode Charset codepages are listed.  There are 162 Charset codepages available on J2SE 1.6.0_16 including 64 valid ASCII and 15 valid EBCDIC codepages.  If the EBCDIC codepage you need is not on the host system, you can replace the EBCDIC codepage name with a file specification such as IBM1047.HCP (example included) which is in hex dump format.  The above defaults match z/OS nnd has been successfully tested on Windows Vista and XP.
```

Comment:
Line 32. Last sentence. "... nnd has been ..." -> "... and has been ...".

Reply:
Text has been edited to remove references to Windows versions and the fact its tested.. that is assumed and does not need to be in the docs.